bin/brew

```diff
@@ -82,11 +94,11 @@ export HOMEBREW_REPOSITORY
 export HOMEBREW_LIBRARY
 export HOMEBREW_CELLAR
 
-if [ -n "$HOMEBREW_OSX" ]
+if [[ -n "$HOMEBREW_OSX" ]]
 then
-  if [ "$('xcode-select' --print-path)" = "/" ]
+  if [[ "$('xcode-select' --print-path)" = "/" ]]
```

Comment:
The few other places I could find that use `xcode-select` refer to it with its full path (`/usr/bin/xcode-select`). Should we do the same here?